cmd/cryptogen/main.go

```diff
@@ -265,6 +280,40 @@ func getConfig() (*Config, error) {
 		return nil, fmt.Errorf("Error Unmarshaling YAML: %s", err)
 	}
 
+	for idx := range config.OrdererOrgs {
```

Comment:
The constant default value is used six times in this block. It needs to be extracted out.
The logic that's in this loop should be extracted to a function that takes a reference to the OrgSpec.